lib/node_modules/@stdlib/stats/base/dsmeanpw/manifest.json

```diff
@@ -25,17 +28,18 @@
   "confs": [
     {
       "task": "build",
+      "wasm": false,
       "src": [
-        "./src/dsmeanpw.c"
+        "./src/main.c"
       ],
       "include": [
         "./include"
       ],
-      "libraries": [
-        "-lm"
-      ],
+      "libraries": [],
       "libpath": [],
       "dependencies": [
+        "@stdlib/blas/base/shared",
+        "@stdlib/strided/base/stride2offset",
         "@stdlib/blas/ext/base/dssum",
```

Comment:
Would you mind updating this dep to explicitly be `dssumpw`, rather than `dssum`? Applies here and below.